### ISSUE TITLE
Link naar service en capabilities bij details geodataset

### ISSUE THREAD

**Issue by gerardnienhuis** [IDgis/geoportaal-test#459]
Nu staat er bij de details van een geodataset wel veel, wat verwijst naar zaken waar het mee te maken heeft. Viewer en download zijn direct aan te klikken en levert dan ook resp. kaartje of downloadscherm.
Maar bij de verwijzing naar de services is dat niet zo.

Kan dat ook aangepast worden? Met verwijzing en doorlink naar de service waarin de betreffende geodataset is opgenomen. En, indien mogelijk, ook met klik op de service een listing van de capabilties. 

Nu staat er:

Download: https://acc-download.geoportaaloverijssel.nl:443/download/83af4df4-662a-4808-aa0b-f2d21a86d818
Website: http://acc-services.geodataoverijssel.nl:80/viewer/layer/B7_test_openbaar/B7_Gem_TEST_openbaar
OGC:WFS: http://acc-services.geodataoverijssel.nl:80/geoserver/B7_test_openbaar/wfs
OGC:WMS: http://acc-services.geodataoverijssel.nl:80/geoserver/B7_test_openbaar/wms
XML: https://acc-intern.geoportaaloverijssel.nl/metadata/dataset/83af4df4-662a-4808-aa0b-f2d21a86d818?noStyle=true

Dat wordt dan:

Download: https://acc-download.geoportaaloverijssel.nl:443/download/83af4df4-662a-4808-aa0b-f2d21a86d818
Viewer: http://acc-services.geodataoverijssel.nl:80/viewer/layer/B7_test_openbaar/B7_Gem_TEST_openbaar
XML: https://acc-intern.geoportaaloverijssel.nl/metadata/dataset/83af4df4-662a-4808-aa0b-f2d21a86d818?noStyle=true

Let op: de tekst Website is veranderd in Viewer

Dan een witregel en daaronder:

Deze dataset is opgenomen in de service B7_test_openbaar

dan weer een witregel, en daaronder

Meer details over deze service: https://acc-intern.geoportaaloverijssel.nl/metadata/service/e80adde2-c081-4e42-95e2-4580272ddd7a (waarbij dan deze url aanklikbaar is)
OGC:WFS: http://acc-services.geodataoverijssel.nl:80/geoserver/B7_test_openbaar/wfs
OGC:WMS: http://acc-services.geodataoverijssel.nl:80/geoserver/B7_test_openbaar/wms

Waarbij dan de url achter de WFS en WMS is een url is, die de capabilities oplevert van de betreffende service.

**Comment by SandroNeumann** [comment]
Website kan niet in viewer veranderd worden, omdat dat niet met de standaard (het protocol van de url) strookt. Ook is er geen metadata element voor de url naar de service metadata.

Linken naar de capabilities documenten kan wel natuurlijk en de regel in welke service de dataset zit ook.

**Comment by gerardnienhuis** [comment]
De wens is om "Viewer" neer te zetten in de stylesheet, waar nu "Website" staat. Aanpassen van de stylesheet kan toch wel?

Er staat nu: 
Website: http://acc-services.geodataoverijssel.nl:80/viewer/layer/B7_test_openbaar/B7_Gem_TEST_openbaar

Dat wordt:
Viewer: http://acc-services.geodataoverijssel.nl:80/viewer/layer/B7_test_openbaar/B7_Gem_TEST_openbaar

**Comment by SandroNeumann** [comment]
Nee, want je kunt er niet vanuit uitgaan dat elke website url een url naar een viewer is.

Dit is een duplicaat van #632

**Comment by gerardnienhuis** [comment]
Wens is om tekst niet in xml aan te passen, maar alleen in de stylesheet omdat nu de tekst onnodig verwarrend is.

**Comment by SandroNeumann** [comment]
Dat had ik begrepen en daar is mijn antwoord hierboven ook op gebaseerd.

**Comment by SandroNeumann** [comment]
De download en viewer link wordt als dusdanig getoond mbv de stylesheet door een prefix met de url te vergelijken.

Dit issue gaat naar milestone 1.5.3 omdat dit na de wijzigingen tbv iso 19115 2.0 uitgevoerd gaat worden.
